src/main/java/io/github/bbortt/event/planner/web/rest/SectionResource.java

```diff
@@ -126,6 +126,21 @@ public SectionResource(SectionService sectionService, ProjectService projectServ
         return ResponseUtil.wrapOrNotFound(section);
     }
 
+    /**
+     * {@code POST /sections/project/:projectId/name-exists} : Whether the given name is still unique in this Project.
+     *
+     * @param projectId the Project identifier.
+     * @param name the value to check.
+     * @return true if the value exists.
+     */
+    @PostMapping("/sections/project/{projectId}/name-exists")
+    @PreAuthorize("@projectService.hasAccessToProject(#projectId, \"" + RolesConstants.ADMIN + "\", \"" + RolesConstants.SECRETARY + "\")")
+    public ResponseEntity<Boolean> isNameExistingInProject(@PathVariable Long projectId, @RequestBody String name) {
+        log.debug("REST request to check uniqueness of name '{}' by projectId : {}", name, projectId);
+        Boolean isUnique = sectionService.isNameExistingInProject(projectId, name);
```

Comment:
ist doch das Gegenteil: in dem Fall nicht unique sondern exists

Reply:
ah stimmt, muss ich anpassen.